lib/redlock/client.rb

```diff
@@ -92,6 +92,10 @@ def lock!(*args)
       end
     end
 
+    def locked?(resource)
+      @servers.map { |s| s.locked?(resource) }.uniq.include?(1)
```

Comment:
Why do you `uniq` here before calling `include?`? That iterates the list twice and copies it once without need.

Additionally, I think I'd prefer it if `RedisInstance.locked?` returned a boolean and this function would read 

```suggestion
      @servers.any? { |s| s.locked?(resource) }
```